FirebaseRemoteConfig/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # v4.4.10
 - [changed] Internal code changes - migrate to using the FIS SDK. (#5096)
+- [changed] Internal code changes - include both CFBundleString and CFBundleShortVersionString in the outgoing fetch requests.
```

Comment:
Not sure that "Internal code changes" should be included on this one since this is a user visible change based on the contents of their provided Info.plist files.

Reply:
was debating this one - the change is not meaningful until backend surfaces the new condition. Will remove it.